packages/autocomplete-js/src/__tests__/positioning.test.ts

```diff
@@ -0,0 +1,163 @@
+import { AutocompletePlugin } from '@algolia/autocomplete-core';
+import { waitFor, getByTestId } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+import { autocomplete } from '../';
+
+import querySuggestions from './fixtures/query-suggestions.json';
+
+type QuerySuggestionFacetMatch = {
+  value: string;
+  count: number;
+};
+type QuerySuggestionsHit = {
+  instant_search: {
+    exact_nb_hits: number;
+    facets: {
+      exact_matches: {
+        categories: QuerySuggestionFacetMatch[];
```

Comment:
these are optional, no?

Reply:
That's the typing for the QS fixture, they all have `categories` in the dataset.